test/utils_test.dart

```diff
@@ -84,6 +84,28 @@ void main() {
     });
   });
 
+  group('combineTypedReducers', () {
+    test('works with inferred types', () {
+      List<TypedReducer<String, dynamic>> getTypedReducers() {
+        return [
+          new TypedReducer<String, TestAction1>(testAction1Reducer),
+          new TypedReducer<String, TestAction2>(testAction2Reducer),
+        ];
+      }
+
+      final store = new Store<String>(
+        combineTypedReducers(getTypedReducers()),
```

Comment:
nitpick: the analyzer is giving a warning, consider changing to

```dart
combineTypedReducers<String, dynamic>(getTypedReducers())
```

Reply:
Done